go/engine/identify.go

```diff
@@ -197,8 +196,13 @@ func (e *Identify) run(ctx *Context) (*libkb.IdentifyOutcome, error) {
 	ctx.IdentifyUI.LaunchNetworkChecks(res.ExportToUncheckedIdentity(), e.user.Export())
 	waiter := e.displayUserCardAsync(ctx)
 
-	e.user.IDTable().Identify(is, e.arg.ForceRemoteCheck, ctx.IdentifyUI, nil)
-	waiter()
+	if err := e.user.IDTable().Identify(is, e.arg.ForceRemoteCheck, ctx.IdentifyUI, nil); err != nil {
+		return nil, err
+	}
+
```

Comment:
This is interesting, I wish Go could do this in a slightly more terse way, but I like your change here.